File: plugin/octo_beat_plugin/android/src/main/kotlin/com/octo/octo_beat_plugin/plugin/event/OctoBeatEventCallback.kt
package com.octo.octo_beat_plugin.plugin.event

import com.octo.octo_beat_plugin.core.device.model.DXHDevice


interface OctoBeatEventCallback {
    fun updateInfo(dxhDevice: DXHDevice)
    fun newMCTEvent(dxhDevice: DXHDevice)
}

File: plugin/octo_beat_plugin/ios/Classes/plugin/event/OctoBeatEventCallback.swift
//
//  OctoBeatEventCallback.swift
//  octo_beat_plugin
//
//  Created by TRAN QUANG DAI on 09/02/2023.
//

import Foundation

protocol OctoBeatEventCallback {
    func updateInfo(device: DeviceDisplay)
    func newMCTEvent(deviceId: UUID)
}
